custom_components/adaptive_cover/translations/en.json

```diff
@@ -38,6 +38,7 @@
         "data": {
           "set_azimuth": "Window Azimuth",
           "window_height": "Window Height",
+          "window_position": "Window Position",
```

Comment:
You should add a longer description of this new parameter in the "data_description" section so that it appears in the HA GUI during configuration. Then users don't have to come to GitHub and scroll through the README to figure out the meaning of this new parameter.